prototypes/component.lua

```diff
@@ -102,8 +102,10 @@ local VALID_POWER_TRANSFER_RATES = {1,2,5,10,20,50,100,200,500,1000,2000,5000,10
 
 function make_energy_interfaces(size,passive_input,passive_output,icon)
 	local j = size/2-0.3
-	local input_priority = (passive_input and "terciary") or "secondary-input"
-	local output_priority = (passive_output and "terciary") or "secondary-output"
+	-- local input_priority = (passive_input and "terciary") or "secondary-input"
+	-- local output_priority = (passive_output and "terciary") or "secondary-output"
+	local input_priority  = "secondary-input"
+	local output_priority = "secondary-output"
```

Comment:
I would prefer if you use the old code and replace `terciary` with `tertiary`. The developers fixed that spelling mistake with the new factorio version.

Reply:
Ah so that was what it was? I couldn't find that in the changelog and figured I'd just try it with secondary-input and secondary-output since @MagmaMcFry  had left a comment that terciary was only being used due to a bug with the secondary-input/outputs. 

That said, I noticed no problems using secondary-input/output so perhaps that bug was fixed?

Reply:
IDK in this case better wait for input from @MagmaMcFry